### ISSUE TITLE
I can't run another training script while one training script is running in keras

### ISSUE THREAD

**Issue by connorchen** [keras-team/keras#7773]
i usually scripting a training code in python with keras to train a model, but when one training script is running, i can not run another training script, why?

such as one running script:
`Epoch 1/20000
62/62 [============================>.] - ETA: 0s - loss: 5.3554 - acc: 0.6588Epoch 00000: val_loss improved from inf to 5.41467, saving model to D:/project/x_ray/model/xray_ver6_color/cc3.hd5
63/62 [==============================] - 159s - loss: 5.3563 - acc: 0.6589 - val_loss: 5.4147 - val_acc: 0.6641
Epoch 2/20000`

when i try to run another script get errors
The following is the error information:
`2017-08-30 11:54:14.759137: I c:\tf_jenkins\home\workspace\release-win\m\windows-gpu\py\35\tensorflow\core\common_runtime\gpu\gpu_device.cc:961] DMA: 0 
2017-08-30 11:54:14.759323: I c:\tf_jenkins\home\workspace\release-win\m\windows-gpu\py\35\tensorflow\core\common_runtime\gpu\gpu_device.cc:971] 0:   Y 
2017-08-30 11:54:14.759522: I c:\tf_jenkins\home\workspace\release-win\m\windows-gpu\py\35\tensorflow\core\common_runtime\gpu\gpu_device.cc:1030] Creating TensorFlow device (/gpu:0) -> (device: 0, name: GeForce GTX 1060 3GB, pci bus id: 0000:01:00.0)
2017-08-30 11:54:28.104755: E c:\tf_jenkins\home\workspace\release-win\m\windows-gpu\py\35\tensorflow\stream_executor\cuda\cuda_dnn.cc:359] could not create cudnn handle: CUDNN_STATUS_NOT_INITIALIZED
2017-08-30 11:54:28.105097: E c:\tf_jenkins\home\workspace\release-win\m\windows-gpu\py\35\tensorflow\stream_executor\cuda\cuda_dnn.cc:366] error retrieving driver version: Unimplemented: kernel reported driver version not implemented on Windows
2017-08-30 11:54:28.105436: E c:\tf_jenkins\home\workspace\release-win\m\windows-gpu\py\35\tensorflow\stream_executor\cuda\cuda_dnn.cc:326] could not destroy cudnn handle: CUDNN_STATUS_BAD_PARAM
2017-08-30 11:54:28.105710: F c:\tf_jenkins\home\workspace\release-win\m\windows-gpu\py\35\tensorflow\core\kernels\conv_ops.cc:671] Check failed: stream->parent()->GetConvolveAlgorithms(&algorithms) 

Process finished with exit code -1073740791 (0xC0000409)`

It seems that a GPU can only perform a training task, doesn't it?
